### ISSUE TITLE
Fatal: number of used blobs is larger than number of available blobs!

### ISSUE THREAD

**Issue by unclesamwk** [restic/restic#1633]
## Output of `restic version`
restic 0.8.2

## How did you run restic exactly?
restic -r /data/server/folder prune

## What backend/server/service did you use to store the repository?
local

## Output
```
password is correct
counting files in repo
building new index for repo
Load(<data/96897d2693>, 591, -3) returned error, retrying after 454.5285ms: offset is negative
Load(<data/96897d2693>, 591, -3) returned error, retrying after 961.13049ms: offset is negative
Load(<data/96897d2693>, 591, -3) returned error, retrying after 1.202589999s: offset is negative
Load(<data/96897d2693>, 591, -3) returned error, retrying after 1.094256014s: offset is negative
Load(<data/96897d2693>, 591, -3) returned error, retrying after 2.834902288s: offset is negative
Load(<data/96897d2693>, 591, -3) returned error, retrying after 3.147825541s: offset is negative
Load(<data/96897d2693>, 591, -3) returned error, retrying after 6.720594448s: offset is negative
Load(<data/96897d2693>, 591, -3) returned error, retrying after 5.092177513s: offset is negative
Load(<data/96897d2693>, 591, -3) returned error, retrying after 15.397977131s: offset is negative
Load(<data/96897d2693>, 591, -3) returned error, retrying after 28.797783124s: offset is negative
pack file cannot be listed 96897d26931a5a4ba3fab5a4e130c9999c76ca3c48cd3776b9ae10efc860f395: offset is negative
[1:05] 100.00%  159 / 159 packs
repository contains 158 packs (25591 blobs) with 560.410 MiB
processed 25591 blobs: 0 duplicate blobs, 0B duplicate
load all snapshots
find data that is still in use for 22 snapshots
[0:02] 100.00%  22 / 22 snapshots
Fatal: number of used blobs is larger than number of available blobs!
Please report this error (along with the output of the 'prune' run) at
https://github.com/restic/restic/issues/new
```

greetings
Sam

**Comment by rawtaz** [comment]
Reference: #1621

**Comment by ifedorenko** [comment]
Does data/96/96897d26931a5a4ba3fab5a4e130c9999c76ca3c48cd3776b9ae10efc860f395 exist in the repository? What is the size of the file, if it does?

**Comment by fd0** [comment]
Oh, this is an interesting bug! Thank you very much for the report! I thought we'd weed out any of those, but apparently we missed one.

Can you please do the following:
 * Please verify that the file `data/96/96897d26931a5a4ba3fab5a4e130c9999c76ca3c48cd3776b9ae10efc860f395`
 * Paste the output of:
```
ls -al data/96/96897d26931a5a4ba3fab5a4e130c9999c76ca3c48cd3776b9ae10efc860f395
stat data/96/96897d26931a5a4ba3fab5a4e130c9999c76ca3c48cd3776b9ae10efc860f395
sha256sum data/96/96897d26931a5a4ba3fab5a4e130c9999c76ca3c48cd3776b9ae10efc860f395
```
 * If the file is small enough, paste the last ~40 lines or so of:
```
xxd data/96/96897d26931a5a4ba3fab5a4e130c9999c76ca3c48cd3776b9ae10efc860f395
```

Thanks!

**Comment by unclesamwk** [comment]
Hey,
sorry, in the battle of the day i've deleted this repo ... ;-( ... in the testing szenario we habe over 7500 repos, When we do rollout it wil be ~150.000 repos. If the error occurs again, I write it in this issue
greetings
Sam

**Comment by fd0** [comment]
Thanks for the report anyway! You found a bug which will be resolved in #1635
